web/less/room.less

```diff
@@ -307,9 +307,8 @@ body {
     }
     a {
       position: absolute;
-      background-color: #0888b3;
       right: 0;
-      border-left: 2px solid #3dc4f0;
+      border-left: 2px solid;
       font-size: 12px;
```

Comment:
border-left is missing color. is this intentional?

Reply:
Yes, I left it like this since i wanted it to be white. Anyway we can explicitly define the color using variables now, so i didnt make sense to put an hex there anyway